### ISSUE TITLE
Generate the QR code to link the proof of co-creation

### ISSUE THREAD

**Issue by AltiMario** [e-Learning-DAO/POCRE#90]
Currently, the QRcode displayed is just a fake, it is just to give an idea of what the user should see, but now it is time for proper implementation.
The QR code is generated ONLY after the user saves the draft. It is a simple representation of the unique link representing the "preview" (rename in "Proof Of co-CREation - Status pending"), where users can see details about the creation, materials used, authors, claiming process (if any), assignation date-time, etc.

![image](https://user-images.githubusercontent.com/598726/197530356-1aa2d39b-1316-4627-bc8c-d7555a0f7100.png)
(not "Creation Authorship Acknowledgement" but "Proof Of co-CREation - Status pending")

Add also the ID in the doc, to facilitate the user who wants to print it.

At some point, in the near future, the "Proof Of co-CREation" info (or just the ID) will be stored also in the blockchain, but it's a ticket for later.

**Comment by huzaifa-99** [comment]
Do we also add the QR code in the preview?

**Comment by AltiMario** [comment]
I think it's not needed inside the doc

**Comment by huzaifa-99** [comment]
@AltiMario Let's say a user printed the doc and wants to view it in the app. A quick QR scan might help opening it (the id will be a little hard since the user will have to map to correct URL). 

Maybe I am overthinking, this could be an enhancement in later stages

**Comment by AltiMario** [comment]
ok, I'm not completely against having it somewhere on the top corner but little in dimension

**Comment by AltiMario** [comment]
An extra requirement. We want also make sure the "proof" data can be stored in a easily manner (in the blockchain for example), so the endpoint `.../12345x/proof` (as an example of the resource 12345x) will return the JSON (mapping the data) but also it will open a basic webpage if called via browser, like this 
![image](https://user-images.githubusercontent.com/598726/198717478-9b476ecd-417d-4728-8c5e-c52d7c9f17fa.png)

**Comment by huzaifa-99** [comment]
Updated it to look a little better
https://pocre-api.herokuapp.com/v1/creations/288524c3-0789-447e-a569-38b9fafaf775/proof
![Screenshot 2022-11-02 at 4 07 55 AM](https://user-images.githubusercontent.com/68777211/199359091-ca75ff68-c3ae-4178-9b8b-1d2d872bf953.png)

**Comment by AltiMario** [comment]
I found an issue with the generated QRcode but I also have a request
![image](https://user-images.githubusercontent.com/598726/199734772-2853a045-33d1-43a6-bf9d-20b757af77b1.png)

I need a new entry in the JSON, do you want me to create a new ticket or add a comment to the previous one (even if it's closed)
![image](https://user-images.githubusercontent.com/598726/199734876-3477a7d2-4507-44e5-b691-59a5929e9ea8.png)

Just for confirmation, all the QR codes point to frontend web app links

**Comment by AltiMario** [comment]
I found an issue with the generated QRcode but I also have a request
![image](https://user-images.githubusercontent.com/598726/199734772-2853a045-33d1-43a6-bf9d-20b757af77b1.png)

I need a new entry in the JSON, do you want me to create a new ticket or add a comment to the previous one (even if it's closed)
![image](https://user-images.githubusercontent.com/598726/199734876-3477a7d2-4507-44e5-b691-59a5929e9ea8.png)

Just for confirmation, all the QR codes point to frontend web app links
